.github/workflows/check-spelling.yml

```
@@ -7,29 +7,33 @@ on:
+                      cspell:bash/dict/bash-words.txt
+                      cspell:filetypes/filetypes.txt
+                      cspell:fonts/fonts.txt
+                      cspell:fullstack/dict/fullstack.txt
+                      cspell:rust/dict/rust.txt
+                      cspell:typescript/dict/typescript.txt
+                      cspell:sql/src/sql.txt
+                      cspell:pt_BR/src/hunspell/index.dic
+                      cspell:pt_BR/src/hunspell/index.aff
+                      cspell:npm/dict/npm.txt
+                      cspell:node/dict/node.txt
+                      cspell:docker/src/docker-words.txt
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)